datadog/provider.go

```diff
@@ -23,17 +23,17 @@ func Provider() terraform.ResourceProvider {
 			"api_key": {
 				Type:        schema.TypeString,
 				Required:    true,
-				DefaultFunc: schema.EnvDefaultFunc("DATADOG_API_KEY", nil),
+				DefaultFunc: schema.MultiEnvDefaultFunc([]string{"DD_API_KEY", "DATADOG_API_KEY"}, nil),
```

Comment:
Nice, TIL `MultiEnvDefaultFunc` I'm torn on which variable should be first here. According to https://github.com/hashicorp/terraform/blob/master/helper/schema/schema.go#L287, the first available variable in the list that has a value is used.

I'm wondering if the safest path is to have the old value (`DATADOG_`) be the first one, so in the case a user has two set, i.e. one for testing and another for another org, we'd choose the old value. WDYT?

Reply:
Definitely agree with you there, just in case customer has both set previously.